deploy/kubernetes/base/controller-serviceaccount.yaml

```diff
@@ -47,7 +47,7 @@ metadata:
 subjects:
   - kind: ServiceAccount
     name: efs-csi-controller-sa
-    namespace: sample
+    namespace: default
```

Comment:
Do we want this?

Reply:
IDK where it came from. it's make generate-kustomize at work.

I think default makes more sense than sample though?

Reply:
Yeah agreed. Just wanted to make sure we don't include something we don't want.